app/controllers/orga/teams_controller.rb

```diff
@@ -89,6 +88,12 @@ def conference_list
     Conference.in_current_season
   end
 
+  def terms_accepted?
```

Comment:
As said in my comment above, I'd suggest to solve this via an acceptance validation - way cleaner in my eyes :v: